docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.zindex.scss

```diff
@@ -1,3 +1,4 @@
+// TODO maybe namespace these?
```

Comment:
We should consider defining a larger z-index scale and then scoping all z-index defs to this file like [here](https://github.com/sparkbox/design-system-starter/blob/master/src/scss/settings/_z-index.scss). The goal would be to only use the `$z-index-` variables in this file so that way we can always see the z-index relationships throughout the file. This prevents the dreaded `z-index: 9999999999999;` rules.